docs/deeplinking.md

```diff
@@ -0,0 +1,180 @@
+
+2. All-in-one headsets do not yet have support for this deeplinking protocol unless invoked through the Nanome in-VR web browser.
+
+## Use cases
+
+- Loading a single file 
+        
+>A scientist visits a web page that is showing a single molecule or project with information about it. Clicking the "Open in Nanome" button would launch Nanome and automatically load in a PDB, Nanome/Pymol session file, or any of our supported file types.
+
+<a href="nanome://eyJjb21tYW5kcyI6W3siam9pbiI6InRoaXMgaXMgbXkgcm9vbSJ9XX0=
+    " class="btn">Open in a single file in Nanome</a>
+
+- Loading Multiple files 
+
+> A scientist visits a web page that has many ligands and a protein. The scientist could then select several compounds and click "Open in Nanome". The Nanome application launches automatically loading in the ligands aligned to the protein.
+
+<a href="nanome://eyJjb21tYW5kcyI6W3siam9pbiI6InRoaXMgaXMgbXkgcm9vbSJ9XX0=
+" class="btn">Open in a Nanome file in Nanome</a>
+
+
+- Run a script
+
+> A scientist can click a button in the web browser and it would would load and run a Nanome Macro (written in LUA) in the room and apply changes such as changing the molecular representation for the secondary structure, doing modifying molecular operations
+
+<a href="nanome://eyJjb21tYW5kcyI6W3siam9pbiI6InRoaXMgaXMgbXkgcm9vbSJ9XX0=" class="btn">Run a script in Nanome</a>
+
+- Load molecule(s) and run a script (example code)
+
+> A scientist visits a web page for a project, selects multiple ligands and a protein, changes some aspect of the molecular representation (e.g. specific surface coloring) and  clicks *Open in Nanome*. The Nanome application launches automatically loading in the ligands, aligning them to the protein and rendering a specific surface coloring.
+
+<a href="nanome://eyJjb21tYW5kcyI6W3siam9pbiI6InRoaXMgaXMgbXkgcm9vbSJ9XX0=
+" class="btn">Open files and run a script in Nanome</a>
+
+## Usage
+
+
+### Constructing the the URL
+
+Each URL contains a JSON stringified, base64 encoded JSON object and is appended to the "nanome" web data/type. 
+
+An example URL looks like:
+    
+     nanome://eyJjb21tYW5kcyI6W3siam9pbiI6InRoaXMgaXMgbXkgcm9vbSJ9XX0=
+
+Entering this url in the browser would automatically launch Nanome from the Windows Registry (PC Setup Installer) or the in-VR web browser would automatically handle the commands.
+
+To create the URL, first create the commands JSON. This one is to load the Protein MMCIF file of PDB: 6LU7 (Covid19 Main Protease) 
+
+
+    const deep_link_commands = [
+        {
+            "type": "load",
+            "sources": [
+                {
+                    "type": "http",
+                    "path": "https://files.rcsb.org/download/6LU7.cif"
+                }
+            ]
+        }
+    ];
+
+Now the JSON needs stringified and base64 encoded (btoa) then executed using the window.open() function: 
+
+    const deep_link = "nanome:" + btoa(JSON.stringify(deep_link_commands));
```

Comment:
Previous examples show `nanome://eyJ...` but this would construct without the `//`, `nanome:eyJ...`. Still valid, but inconsistent